Apps/WebClient/src/ClientApp/src/components/MultiSelectComponent.vue

```diff
@@ -87,7 +87,7 @@ export default class MultiSelectComponent extends Vue {
                             :disabled="disabled"
                             :title="tag"
                             :pill="true"
-                            variant="danger"
+                            variant="light"
```

Comment:
The `pill` property should also be removed.  (I think there are 3 places this needs to be changed.)

Reply:
Only one 'pill' property found in file.  Property removed.